src/routes/eventRouter.js

```diff
@@ -6,48 +6,61 @@ const { getDatabase } = require('../database/mongo');
 
 // GET JSON of all events
 routes.get('/all', async (req, res, next) => {
-
-    Event.find(function (err, events) {
-        if (err) {
-          console.log('error ', err.res);
-        }
-        else {
-          res.status(200).json(events);
-        }
-      })
+  Event.find(function(err, events) {
+    if (err) {
+      console.log('error ', err.res);
+    } else {
+      res.status(200).json(events);
+    }
+  });
 });
 
-
 // POST a new event to the database
 routes.post('/event', (request, res) => {
-    if (!request.body || request.body == {}){
-      return res.send("no request body")
-    }
-    var event = new Event({
-        title: request.body.title,
-        location: request.body.location,
-        startTime: request.body.startTime,
-        endTime: request.body.endTime,
-        checkIn: request.body.checkIn,
-        url: request.body.url
-    }).save((err, response) => {
-        if (err) res.status(400).send(err)
-        res.status(200).send(response)
-    })
-    
+  if (!request.body || request.body == {}) {
+    return res.send('no request body');
+  }
+  var event = new Event({
+    title: request.body.title,
+    location: request.body.location,
+    startTime: request.body.startTime,
+    endTime: request.body.endTime,
+    checkIn: request.body.checkIn,
+    tags: request.body.tags,
+    url: request.body.url
+  }).save((err, response) => {
+    if (err) res.status(400).send(err);
+    res.status(200).send(response);
+  });
 });
 
-// GET specific events from the database via a "query" (title search, description, time, etc.)
-routes.get("/event/:query", (request, res) => {
-    var event = Event.find({ $text: { $search: request.params.query}}, function(err, events){
-      if (err) {
-        console.log('error ', err);
-      }
-      else {
-        res.status(200).json(events);
-      }
-    });
-})
+// @route    DELETE event
+// @desc     Delete an event
+// @access   Private
+router.delete('/', async (req, res) => {
```

Comment:
"router.delete" needs to be "routes.delete" because the Router variable is named "routes." Similar to previous routes created, such as the GET route: `routes.get("/event/:query", (request, res) => {`.

Also do we want the delete route to match a specific url? Right now it points to /events/. Maybe we can combine it with the POST/GET routes on /events/event?